packages/react/src/components/Toggle/index.js

```diff
@@ -5,5 +5,25 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-export * from './Toggle.Skeleton';
-export default from './Toggle';
+import React from 'react';
+
+import ToggleNext from './next/Toggle';
+import ToggleClassic from './Toggle';
+
+import { enabled } from '@carbon/feature-flags';
+import { useFeatureFlag } from '../FeatureFlags';
+
+function Toggle(props) {
+  const enabled = useFeatureFlag('enable-2021-release');
+  if (enabled) {
+    return <ToggleNext {...props} />;
+  }
+  return <ToggleClassic {...props} />;
+}
+
+Toggle.propTypes = enabled('enable-2021-release')
```

Comment:
I think we'll have to defer to the prop type checking of `ToggleNext` / `ToggleClassic` themselves since the `enabled` function returns the global flag value versus the value from the given React tree 😞

Reply:
Oh, I see. I removed it now but the PublicAPI test seems to be confused by it:
https://github.com/carbon-design-system/carbon/pull/8469/files#diff-2cf7ebc90d6f04af3d6e87a1e9de5fc56db503d4c59ca34b1c4451e8b7bac0c9L6125-L6178

Is that ok?